Vision doc (#86)
SpokaneTech/SpokaneTech_Py: .github/VISION.md

> # About the Spokane Tech Project

Author: sure

> ## What is?

Author: I decided to just remove the subtitle

### Phase Two:
Have event requests and suggestions capabilities. Here members can post a suggested events they want to give or have someone else give, and others can up/down vote the event (think reddit or stackoverflow). This can be used to prioritize events base on community interest. This can also serve as a living backlog of event ideas. Add lebels to events, such as technical areas (frontend, scripting, ML, etc.) and topic levels (beginner/intermediate/etc.). With labels people can filter event based on interest and other criteria.

### Phase Three:
Build member profiles. With profiles, we can have some basic metrics on things like career level, geographic location, interested and expertise. This data can help provide visbility into the overall tech presence in Spokane and help drive event topics and location. This could also be a future resouce to make available to local businesses and the communiity for things like contract work, etc. (There has been some outside interest in this type of resoiurce)

### Future goals:  

> The Spokane Tech project was started mostly by members of the Spokane Python User Group (SPUG), so natually the first version of the website is based on python. In the future the project will be re-created in other langauges/frameworks/etc. (such as Golang or Rust) as member interest dictates. This is intended to foster growth, knowledgesharing, and exposure to different tech stacks and methodologies.

Reviewer: ```suggestion
The Spokane Tech project was started mostly by members of the Spokane Python User Group (SPUG), so naturally the first version of the website is based on Python. In the future the project may be re-created in other languages/frameworks/etc. (such as Go or Rust) as member interest dictates. This is intended to foster growth, knowledge sharing, and exposure to different tech stacks and methodologies.
```

## Interested in participating? Great! Read on...

Here are a few things you can do to get started.

> - Go to the site and explore. It's live at https://www.spokanetech.org

Reviewer: ```suggestion
- Go to the site and explore. It's live at https://spokanetech.org
```

- Look through the open issues and find one that interests you (issues tagged "good first issue" could be great candidates)
https://github.com/SpokaneTech/SpokaneTech_Py/issues 

- Read the docs: In our documentation directory we have design decisions, style guide, contributing guide and more. Feel free to familiarize yourself with these. 
https://spokanetech.github.io/SpokaneTech_Py/ 

> - clone the repo to you machine and run locally, explore the code, break things, fix things, have fun. Step by step instructions are in the CONTRIBUTION doc.

Reviewer: ```suggestion
- Clone the repo to you machine and run locally, explore the code, break things, fix things, have fun. Step by step instructions are in the [CONTRIBUTING](https://spokanetech.github.io/SpokaneTech_Py/CONTRIBUTING/) doc.
```
